getlantern/replica
# [replica:server] Use infohash instead of UUID for upload artifacts

Test it like this:

- Clone lantern-desktop and replica
- Add this to lantern-desktop's `go.mod`: `replace github.com/getlantern/replica => ../replica`
- Go to your recently-cloned local `replica` repo. Check out this PR's branch in your local `replica` repo
- Go back to `lantern-desktop`.
- Run `cd desktop/replica; go test -race -run TestUploadAndDelete`. Test should PASS
